docs/content/author-apps/cloud-resources/azure-resources/snippets/azure-connection.bicep

```
@@ -1,6 +1,8 @@
 import radius as radius
 
 param environment string
+
+@description('The geo-location where the resource lives.')
```

Let's do something like:

```suggestion
@description('The Azure region to deploy Azure resource(s) into. Defaults to the region of the target Azure resource group.')
```

`geo-location` isn't a well-defined term.